lib/timesTable.js

```diff
@@ -0,0 +1,16 @@
+var timesTable = function(n) {
```

Comment:
I know you're learning Go, so I imagine that may be why you're using letters as variables. In JS, it's not cool, Glen CoCo